resources/wren/shaders/phong.frag

```diff
@@ -196,7 +196,7 @@ void main() {
       if (material.textureFlags.z == 0.0)
         texColor = mainColor;
       else
-        texColor = vec4(mix(texColor.xyz, mainColor.xyz, mainColor.w), texColor.w + mainColor.w);
+        texColor = vec4(mix(texColor.xyz, mainColor.xyz, mainColor.w), clamp(texColor.w + mainColor.w, 0, 1));
```

Comment:
This was the cause of the `display_attach_camera` test failure, in some case the alpha was bigger than 1.
This is probably ok for NVIDIA drivers but mesa doesn't seem to handle this well.